core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -1181,6 +1181,7 @@ public void updateMetadataVersion() {
             throwDistressException(err);
         }
 
+        bumpStructureVersion();
```

Comment:
we need to bump TXN, not structure version.

Structure version is for when table structure changes - column added or removed etc.

TXN is for when data changed. NULL field qualifies because NULL can be added in due course of data ingestion